pkg/simulate/retrieval/retrieval.go

```diff
@@ -47,11 +44,13 @@ func NewDefaultOptions() Options {
 var _ beekeeper.Action = (*Simulation)(nil)
 
 // Simulation instance
-type Simulation struct{}
+type Simulation struct {
+	metrics metrics
+}
 
 // NewSimulation returns new upload simulation
 func NewSimulation() beekeeper.Action {
-	return &Simulation{}
+	return &Simulation{newMetrics("")}
```

Comment:
As all metrics are collected to the single database and the pusher is used which makes no initial values push, the previous and next runs are connected in the graph as the last value from the previous run is actually the first one from the next one. This is misleading and separating runs with label helps identifying the correct values for a particular simulation run.

This can be reintroduced later, again.

Reply:
:+1: the reason I left it the way it is is that I did not want to pollute the `NewSimulation` ctor with data like the cluster name. I think that once this is necessary we could find some other way to create the differentiation, maybe with some just the unix epoch when the job was started, or something along those lines, since the previous ctor had the cluster name in it